py-polars/polars/functions/lazy.py

```diff
@@ -319,31 +477,48 @@ def approx_n_unique(column: str | Expr) -> Expr:
     ╞═════╡
     │ 2   │
     └─────┘
+    >>> df.select(pl.approx_n_unique("b", "c"))
+    shape: (1, 2)
+    ┌─────┬─────┐
+    │ b   ┆ c   │
+    │ --- ┆ --- │
+    │ u32 ┆ u32 │
+    ╞═════╪═════╡
+    │ 3   ┆ 2   │
+    └─────┴─────┘
 
     """
-    if isinstance(column, pl.Expr):
-        return column.approx_n_unique()
```

Comment:
I guess this is technically breaking, but yeah let's get rid of it since it's the odd one out here.